ansible/roles/hyrax/tasks/compel.yml

```diff
@@ -0,0 +1,44 @@
+---
+- block:
+  - name: load workflow
+    command: ./bin/rails hyrax:workflow:load
+    args:
+      chdir: '{{ project_app_root }}'
+
+  - name: create default collection type
+    command: ./bin/rails compel:create_default_collection_type
+    args:
+      chdir: '{{ project_app_root }}'
+
+  - name: create roles
+    command: ./bin/rails compel:add_roles
+    args:
+      chdir: '{{ project_app_root }}'
+
+  - name: ensure font directory exists
+    file:
+      path: '{{ project_app_root }}/app/assets/fonts'
+      state: directory
+
+  - name: find old font files
+    find:
+      path: '{{ project_app_root }}/app/assets/fonts/'
+    register: fonts
+
+  - name: delete old font files
+    file:
+      path: '{{ item.path }}'
+      state: absent
+    with_items: '{{ fonts.files }}'
+    notify: clear rails cache
+
+  - name: add new font files
+    unarchive:
+      src: '{{ local_files_dir }}/fonts.zip'
+      dest: '{{ project_app_root }}/app/assets/fonts/'
+    ignore_errors: True
+    notify: clear rails cache
+  become: yes
+  become_user: '{{ project_owner }}'
```

Comment:
Rake tasks affecting internal application state should usually be executed as `project_runner` whereas file system operations (like copying files, making directories, and unpacking .zip files) should be executed as `project_owner` so as to better observe permissions.  See, e.g., the `hyrax/tasks/iawa.yml` local tasks plays for an example of this.  (It uses different users across different blocks.)